modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -409,7 +412,11 @@ public void testUpdateUserDefinedAuthenticatorOfIdP() throws JsonProcessingExcep
                 .body("name", equalTo(new String(Base64.getDecoder().decode(FEDERATED_AUTHENTICATOR_ID))))
                 .body("isEnabled", equalTo(true))
                 .body("definedBy", equalTo(DEFINED_BY_USER))
-                .body( "tags", Matchers.hasItems("Custom"))
+                .body( "tags", hasItems("Custom"))
+                .body("self", equalTo(getTenantedRelativePath(
```

Comment:
Update response does not have self, ot has only endpoint config.

Reply:
This is updated with 19889d29a846f9073e796922aee6f22da20d7303